source/concepts/networking/network-firewall.html.md.erb

```diff
@@ -0,0 +1,56 @@
+---
+owner_slack: "#modernisation-platform"
+title: AWS Network Firewall
+last_reviewed_on: 2023-02-08
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+We currently have setup in our core-network account AWS Network Firewall to monitor and control network traffic on a layer 3 level for transit gateway connection and cross account VPCs to protect resources
```

Comment:
Technically speaking this will be inaccurate after this sprint - we're going to start reading FQDNs which is going to be a layer 7 function (hostname in a HTTP header)

Reply:
updated so that it mentions layer 3-7